votor/src/common.rs

```diff
@@ -37,16 +37,65 @@ pub const fn conflicting_types(vote_type: VoteType) -> &'static [VoteType] {
 /// Must be in sync with `vote_to_cert_types`
 pub const fn certificate_limits_and_vote_types(
     cert_type: &CertificateType,
-) -> (f64, &'static [VoteType]) {
+) -> (Fraction, &'static [VoteType]) {
     match cert_type {
-        CertificateType::Notarize(_, _) => (0.6, &[VoteType::Notarize]),
-        CertificateType::NotarizeFallback(_, _) => {
-            (0.6, &[VoteType::Notarize, VoteType::NotarizeFallback])
+        CertificateType::Notarize(_, _) => (Fraction::new(3, 5), &[VoteType::Notarize]),
```

Comment:
can we have a constructor that takes a percentage, `Fraction::new(3, 5)` is much less than clear than something like `Fraction::new(60)